<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoAddTagsDialog</class>
 <widget class="QDialog" name="GoAddTagsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Tags To Struct Field</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="jsonCheckBox">
     <property name="text">
      <string>Add JSON Tag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jsonGroupBox">
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="jsonOptionsCheckBox">
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jsonOptionsLineEdit">
        <property name="toolTip">
         <string>Multiple options separated by commas</string>
        </property>
        <property name="text">
         <string notr="true">omitempty,</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="xmlCheckBox">
     <property name="text">
      <string>Add XML Tag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="xmlGroupBox">
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="xmlOptionsCheckBox">
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xmlOptionsLineEdit">
        <property name="toolTip">
         <string>Multiple options separated by commas</string>
        </property>
        <property name="text">
         <string notr="true">omitempty,</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="customCheckBox">
     <property name="text">
      <string>Add Custom Tags</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customGroupBox">
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Tag Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customTagNameLineEdit1">
          <property name="toolTip">
           <string>Setup custom tag name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Options</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customTagOptionLineEdit1">
          <property name="toolTip">
           <string>Multiple options separated by commas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tag Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customTagNameLineEdit2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Options</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customTagOptionlineEdit2">
          <property name="toolTip">
           <string>Multiple options separated by commas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string notr="true">Ttransform</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="snakeCaseRadioButton">
        <property name="text">
         <string notr="true">SnakeCase</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="camelCaseRadioButton">
        <property name="text">
         <string notr="true">CamelCase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lispCaseRadioButton">
        <property name="text">
         <string notr="true">LispCase</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>9</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="transformInfo">
        <property name="text">
         <string notr="true">BaseDomain -&gt; base_domain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="sortCheckBox">
        <property name="text">
         <string>Sort sorts the tags in increasing order according to the key name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overrideCheckBox">
        <property name="text">
         <string>Override current tags when adding tags</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="argumentsEdit"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GoAddTagsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GoAddTagsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
